src/main/pages/che-7/administration-guide/proc_configuring-prod-short-to-use-external-keycloak.adoc

```diff
@@ -38,7 +38,7 @@ on OpenShift
 endif::[]
 and uses the default OpenShift OAuth support, user authentication relies on the integration of {identity-provider} with OpenShift OAuth. This allows users to log in to {prod-short} with their OpenShift login and have their workspaces created under personal OpenShift namespaces. 
 +
-This requires setting up an OpenShift identity provider inside {identity-provider}. When using an external {identity-provider}, set up the identity provider manually. For instructions, seethe appropriate {identity-provider} documentations for either link:https://www.keycloak.org/docs/latest/server_admin/#openshift-3[OpenShift 3] or link:https://www.keycloak.org/docs/latest/server_admin/#openshift-4[OpenShift 4].
+This requires setting up an OpenShift identity provider inside {identity-provider}. When using an external {identity-provider}, set up the identity provider manually. For instructions, seethe appropriate {identity-provider} documentations for either link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html/server_administration_guide/identity_broker#openshift_3[OpenShift 3] or link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.4/html/server_administration_guide/identity_broker#openshift_4[OpenShift 4].
```

Comment:
@MichalMaler We should have both URLs here, and have conditionals that drive whether we put either the upstream or the downstream URLs

Reply:
The correct way to fix this problem is to add keys to `_config.yml` as in https://github.com/eclipse/che-docs/pull/1423